publication-event-handlers/src/main/java/no/unit/nva/publication/events/handlers/tickets/AcceptedPublishingRequestEventHandler.java

```diff
@@ -158,64 +181,121 @@ private RuntimeException throwException(String message, Failure<?> failure) {
     }
 
     private void publishPublication(PublishingRequestCase latestUpdate, UserInstance userInstance) {
-        attempt(() -> resourceService.publishPublication(userInstance, latestUpdate.getResourceIdentifier()))
-            .orElseThrow(fail -> throwException(
-                String.format(PUBLISHING_ERROR_MESSAGE, latestUpdate.getResourceIdentifier()), fail));
+        attempt(
+                        () ->
+                                resourceService.publishPublication(
+                                        userInstance, latestUpdate.getResourceIdentifier()))
```

Comment:
Is it google code style, or stil unitStyle?

Reply:
I have the google code style enabled. Surprised by the amount of changes it make to the formatting.

Reply:
Should we add config for codestyle @LarsV123 implemented in search-api here as well? Cause for now the most of us still use NVA code style and not google code style.